services/QuillLMS/app/models/blog_post.rb

```diff
@@ -1,7 +1,7 @@
 class BlogPost < ActiveRecord::Base
-  TOPICS = ['Getting Started', 'Teacher Stories', 'Writing Instruction Research', 'Announcements', 'Press', 'Case Studies', 'Teacher Materials', 'Best Practices', 'Support', 'Webinars', 'Twitter Love', 'Video Tutorials']
+  TOPICS = ["Getting Started", "Teacher Stories", "Writing Instruction Research", "Announcements", "Press", "Case Studies", "Teacher Materials", "Best Practices", "Support", "Webinars", "Twitter Love", "Video Tutorials", "What's New"]
   STUDENT_TOPICS = ['Student Getting Started', 'Student How To']
-  TOPIC_SLUGS = TOPICS.map { |topic| topic.downcase.gsub(' ','-') }
+  TOPIC_SLUGS = TOPICS.map { |topic| topic.downcase.gsub(' ','-').gsub("'", "%27") }
```

Comment:
What do you think about using a general URI-encoding function like `CGI::escape( ... )` instead of explicitly encoding `'` characters?  That way if we add other stuff that needs encoding later, we won't have to make changes.